src/test/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImplTest.java

```diff
@@ -658,10 +665,13 @@ void getTranscriptionThrowsNotFoundExceptionIfTranscriptionDocumentHiddenAndUser
         var transcriptionId = 1;
         var transcription = new TranscriptionEntity();
         transcription.setIsCurrent(true);
+        transcription.setId(transcriptionId);
         var transcriptionDocument = new TranscriptionDocumentEntity();
         transcriptionDocument.setHidden(true);
         transcription.setTranscriptionDocumentEntities(List.of(transcriptionDocument));
         when(mockTranscriptionRepository.findById(transcriptionId)).thenReturn(Optional.of(transcription));
+        when(mockTranscriptionDocumentRepository.findByTranscriptionIdAndHiddenTrueIncludeDeleted(anyInt()))
+            .thenReturn(List.of(new TranscriptionDocumentEntity()));
```

Comment:
Should add verification to ensure this got called and was give the correct params

Reply:
done